components/dashboard/src/data/prebuilds/prebuild-logs-emitter.ts

```diff
@@ -199,6 +210,10 @@ function streamPrebuildLogs(
                 return;
             }
         } catch (err) {
+            if (err instanceof DOMException && err.name === "AbortError") {
+                console.debug("aborted watching headless logs");
```

Comment:
This message is confusing to me, because it does the branch seems to be doing sth different (ignore the error). Can we make that clearer e.g. by adjusting the message? :thinking: